src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -0,0 +1,90 @@
+package org.launchcode.liftoffrecipetracker.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+@Entity
+public class Recipe extends AbstractEntity {
+
+	private HashMap<String, String> ingredient;
```

Comment:
I think your original thought that this should be "ingredients" was right! Very rarely would we find a one-ingredient recipe (I think? I don't know, I could be projecting haha).